services/site/src/graphql/queries/topThreeChallenges.graphql

```diff
@@ -0,0 +1,33 @@
+
+query topThreeChallenges {
+    challenge1: challengeBySlug(slug: "a-valid-html-document") {
+      id
+      slug
+      name
+      status
+      difficulty
+      numberOfLevels
+      numberOfFinishedLevels
+      isMobileFriendly
```

Comment:
The fields should be moved to a fragment to avoid duplication. See https://github.com/a11yphant/a11yphant/blob/c79a47c61c5b3fed3bf27f970a156cd918f1d05b/services/site/src/graphql/queries/challenges.graphql for how to do that